tools/IceRpc.ProtocGen/StringExtensions.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) ZeroC, Inc.
+
+namespace IceRpc.ProtocGen;
+
+internal static class StringExtensions
+{
+    /// <summary>Removes consecutive empty lines, and empty lines that are between an open and a close brace '{', '}'
```

Comment:
As far as I can tell, the only superfluous empty lines we currently generate are:
```
 -- extra empty line here
}
```

And an empty line before a closing } is always incorrect. So can we simplify this code?

Reply:
You are right, I simplified the code.